aniview V3 support

itaynave opened the pull request:
> * [x]  Refactoring (no functional changes, no api changes)
> * [x]  Feature
Remove unsupported v3 function
CCPA support

itaynave commented:
Why is it failing on openX checks? Not related to this PR...

  openx analytics adapter
    sending analytics event
      ✗ should send 1 request to the right endpoint
	expected 'https://ads.openx.net:443/w/1.0/pban' to equal 'https://ads.openx.net/w/1.0/pban'
	Error

Fawke commented on modules/aniviewBidAdapter.js:
```diff
@@ -0,0 +1,199 @@
+  for (let i = 0; i < validBidRequests.length; i++) {
+    let bidRequest = validBidRequests[i];
+
+    if (!bidRequest.sizes || !bidRequest.sizes.length) {
```
**bidRequest.sizes** may be deprecated in the next major release. Alternative is to use the new **bidRequest.mediaTypes** object.

Fawke commented:
Hi @itaynave,

This PR looks fine. 

- I would recommend you to open the same PR against the legacy branch `2.44.x-legacy`.

- Also, can you please open a Docs PR on `prebid.github.io` stating that you now support CCPA so that it's easy for publishers checking out the Docs to know your support CCPA as well. Just add `usp_supported: true` in the header section of the md file.

- Few observations in the md file, you've stated the test parameters like this:

```
var videoAdUnit = [
{
    code: 'video1',
    sizes: [
        [300, 250],
        [640, 480]
    ],
    bids: [{
        bidder: 'aniview',
        params: {
            AV_PUBLISHERID: '55b78633181f4603178b4568',
            AV_CHANNELID: '55b7904d181f46410f8b4568'
        }
    }]
}];
```
Few things to note about this are:
- **sizes** property is deprecated now. Publishers now have to use the **mediaTypes.banner.sizes** to declare the sizes. Or in the case of video, they have to use **mediaTypes.video.playerSize** to declare the playerSize.
- Is there is any reason to omit the **context** field here. The **context** field indicates if the request is for a **outstream**, **instream** or **adpod** type video.

itaynave commented:
- Will open additional PR to V2 once this one is approved
- Opened a doc PR for CCPA
- Updated the sample md
- Fixed support for both playerSize and size
- context is added in sample

bretg commented:
docs PR https://github.com/prebid/prebid.github.io/pull/1716

itaynave commented:
Hi @Fawke 
Thanks for your comments, fixed everything
Let me know if there is anything else

Fawke commented:
@itaynave Thanks for making the requested changes. I plugged your instream params that you've shared in one of the hello_world examples for instream, but I didn't get any response back from the server. Will the params that you've shared in the md file work in an test example. I just wanted to doublecheck the response coming back from the server is fine.

itaynave commented:
Usually each of our partners is getting his own id...
Anyway updated sample in md to a working one.

Fawke commented:
Thanks @itaynave for providing the test parameter. I'm accepting your change in Prebid v3, I'd request you to open a PR against the legacy branch at `2.44.x-legacy`.
